processor/marketplace_processor/offer/offer_acceptance.py

```diff
@@ -243,6 +253,19 @@ def _has_rule(rules, rule_type):
     return False
 
 
+def _accounts_limited_to(offer):
+    if _has_rule(offer.rules, rule_pb2.Rule.EXCHANGE_LIMITED_TO_ACCOUNTS):
+        return True
+    return False
+
+
+def _accounts(offer):
+    return set.intersection(
```

Comment:
I'd guess this is the line causing the error.

Reply:
I was not able to reproduce. You may have been testing an old branch as this issue was fixed earlier in the day.